app/components/model-config.tsx

```
@@ -61,13 +62,14 @@ export function ModelConfigList(props: {
           step="0.1"
           onChange={(e) => {
             props.updateConfig(
-              (config) =>
+              config =>
                 (config.temperature = ModalConfigValidator.temperature(
                   e.currentTarget.valueAsNumber,
                 )),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)